cocos/core/math/math-native-ext.ts

```diff
@@ -56,13 +55,13 @@ export enum MathType {
 }
 
 function extendType (proto: any, parentProto: any, typ: MathType): void {
-    proto._data = function (): Float32Array {
+    proto._data = function _data (): Float32Array {
```

Comment:
It think it is better to be able to use anonymous function.

Reply:
No, here we should not , since ()=> will capture `this` of current context, while here this is object itself, not the context.

Reply:
Got it.